package.json

```diff
@@ -196,7 +196,9 @@
     "pkg-up": "^3.1.0",
     "portfinder": "^1.0.32",
     "prettier": "^2.5.1",
-    "prisma": "^4.5.0",
+    "prisma": "^5.12.1",
+    "prisma-v4": "npm:prisma@4.16.2",
+    "prisma-v450": "npm:prisma@4.5.0",
```

Comment:
4.5 -> support for getConfig
4.16.2 -> broken (cant fix)
5.12.1 -> added new support for replacement of getConfig

Reply:
See comments on PR body